### PR TITLE
Update package.json and code

### PR DESCRIPTION
**Pull request by dhavalsoni9989**
- chore : Added the @lab/moleculer code and package.json file

### CODE REVIEW

**Review by HazemKhaled — CHANGES_REQUESTED** [review]
Kindly add

- [ ] The new test folder structure including integration and unit, supertest npm and env file
- [ ] Add example file for integration test for only one endpoint

File: package.json [hunk 1]
```diff
@@ -39,11 +45,15 @@
...
+    "@moleculer/lab": "^0.6.0",
+    "ioredis": "^4.19.2",
     "lodash": "^4.17.20",
     "moleculer": "^0.14.11",
     "moleculer-db": "^0.8.10",
     "moleculer-db-adapter-mongo": "^0.4.10",
     "moleculer-web": "^0.9.1",
-    "mongoose": "^5.10.11"
+    "mongoose": "^5.10.11",
+    "supertest": "^6.0.1"
```
Comment (HazemKhaled) [inline package.json:57]: Test NPMs should be in devDependencies only

**Review by HazemKhaled — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (dhavalsoni9989) [reply package.json]: Moved suprtest package to devDependencies

Comment (dhavalsoni9989) [reply package.json]: Moved @jest/types to devDependencies

**Review by HazemKhaled — APPROVED** [review]

**Pull request merged**